### PR TITLE
Moves to GitHub actions build (#188)

### PR DESCRIPTION
**Pull request by adriancole**
* Travis is still cutoff despite over a week of email.
 * We have to switch to GitHub actions to avoid paralyzing the project

As a part of this, the build is refactored to use common scripts,
documented in build-bin/README.md

This also re-organizes the directory structure so that parallel tests
work. For example, aggregator sub-poms don't help as plain directories
organize without complicating the graph.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,30 +15,40 @@ which build off interfaces provided by the [zipkin-reporters-java](https://githu
 and [Collectors](https://github.com/openzipkin/zipkin/blob/master/zipkin-collector/core/src/main/java/zipkin2/collector/Collector.java),
 which are used in a [Zipkin](https://github.com/openzipkin/zipkin) server.
 
+We also have tracing libraries that extend [Brave](https://github.com/openzipkin/brave).
+
+## Tracing libraries
```
Comment (adriancole) [inline README.md:20]: formerly not mentioned

File: collector/sqs/src/test/java/zipkin2/collector/sqs/ITSQSCollector.java [hunk 2]
```diff
@@ -34,13 +34,13 @@
...
 
+import static java.nio.charset.StandardCharsets.UTF_8;
 import static java.util.Arrays.asList;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.awaitility.Awaitility.await;
 
 public class ITSQSCollector {
-
-  @Rule public AmazonSQSRule sqsRule = new AmazonSQSRule().start(9324);
+  @Rule public AmazonSQSRule sqsRule = new AmazonSQSRule().start();
```
Comment (adriancole) [inline collector/sqs/src/test/java/zipkin2/collector/sqs/ITSQSCollector.java:43]: this allows parallel tests to work

File: module/README.md [hunk 3]
```diff
@@ -0,0 +1,278 @@
+# module-aws
```
Comment (adriancole) [inline module/README.md:1]: aggregated and deduped (not fun, but worth it)

File: module/src/main/resources/zipkin-server-aws.yml [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# When enabled, this allows shorter env properties (ex -Dspring.profiles.active=aws)
+spring:
+  main:
+    # ZipkinElasticsearchAwsStorageModule overrides the bean named 'storage'
+    allow-bean-definition-overriding: true
+zipkin:
+  internal:
+    module:
+      elasticsearch-aws: zipkin.module.storage.aws.elasticsearchZipkinElasticsearchAwsStorageModule
```
Comment (adriancole) [inline module/src/main/resources/zipkin-server-aws.yml:9]: these are still broken out, just for convenience. the profile itself is no longer sensitive to contents.

**Review by adriancole — COMMENTED** [review]
notes

**Review by anuraaga — APPROVED** [review]

**Pull request merged**